mdformat/_cli.py

```diff
@@ -23,9 +29,11 @@ def run(cli_args: Sequence[str]) -> int:  # noqa: C901
     try:
         file_paths = resolve_file_paths(args.paths)
     except InvalidPath as e:
-        sys.stderr.write(f'Error: File "{e.path}" does not exist.\n')
-        return 1
+        parser.error(f'File "{e.path}" does not exist.\n')
```

Comment:
Not sure but I would guess the newline at the end here is not needed now? (I.e. `parser.error` would act more like a `print()` than a `write()`)